seqfold/seqfold.go

```diff
@@ -0,0 +1,95 @@
+package seqfold
+
+import (
+	"fmt"
+	"math"
+)
+
+type Comp map[byte]byte
+
+type MultiBranch struct {
+	A, B, C, D float64
+}
+
+// Energy holds two energies, enthaply and entropy
+// SantaLucia & Hicks (2004), Annu. Rev. Biophys. Biomol. Struct 33: 415-440
+type Energy struct {
+	// enthapl
+	H float64
+	// entropy
+	S float64
+}
+
+type BpEnergy map[string]Energy
+
+type LoopEnergy map[int]Energy
+
+type Energies struct {
+	BULGE_LOOPS     LoopEnergy
+	COMPLEMENT      Comp
+	DE              BpEnergy
+	HAIRPIN_LOOPS   LoopEnergy
+	MULTIBRANCH     MultiBranch
+	INTERNAL_LOOPS  LoopEnergy
+	INTERNAL_MM     BpEnergy
+	NN              BpEnergy
+	TERMINAL_MM     BpEnergy
+	TRI_TETRA_LOOPS BpEnergy
+}
+
+type IJ struct {
+	I, J int
+}
+
+// A single structure with a free energy, description, and inward children
```

Comment:
think there's a more descriptive name than, "Struct"? Little too similar to struct keyword.

Reply:
Agreed, it tried to keep the code as close to the python original but this is really confusing.

`NucleicAcidStructure` would be the most appropriate name IMHO even if it's a bit long.
I'll make this change but I'm open to suggestion in case you were thinking on a better name.

Reply:
I think long names are okay as long as they tell the story right

Reply:
@folago I'd say it's "done" and ready to be refactored to be idiomatic!